### PR TITLE
add ExlucdeFromLatest field to TargetRegion parameter New/Update-AzGalleryImageVersion

### PR DESCRIPTION
**Pull request by grizzlytheodore**
## Description

Updating Bug: https://msazure.visualstudio.com/One/_workitems/edit/26504946
ICM: https://portal.microsofticm.com/imp/v5/incidents/details/459937637/summary

## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by grizzlytheodore** [comment]
Testing Results: 
image
image
image
image
image

**Comment by grizzlytheodore** [comment]
please review/merge @Nickcandy

File: src/Compute/Compute/ChangeLog.md [hunk 1]
```diff
@@ -32,6 +32,7 @@
   `Get-AzVmss` will no longer allow empty values to `ResourceGroupName` and `VMScaleSetName` to avoid a bug where it will just return nothing.
 * Added a new parameter `-SharingProfile` to `New-AzCapacityReservationGroup` and `Update-AzCapacityReservationGroup`.
 * Added the new parameter `SourceImageVMId` to the `New-AzGalleryImageVersion` cmdlet. Also added some error messages for this new parameter and the existing parameter `SourceImageId`. 
+* Parameter `-TargetRegion` in `New-AzGalleryImageVersion` and `Update-AzGalleryImageVersion` will now accept `ExcludeFromLatest` field.
```
Comment (Nickcandy) [inline src/Compute/Compute/ChangeLog.md:35]: Please rephrase this sentence in the past tense, like the other changelog entries.

File: src/Compute/Compute/Generated/GalleryImageVersion/GalleryImageVersionCreateOrUpdateMethod.cs [hunk 2]
```diff
@@ -125,6 +125,7 @@ public override void ExecuteCmdlet()
                                 Name = (string)t["Name"],
                                 RegionalReplicaCount = (int?)t["ReplicaCount"],
                                 StorageAccountType = (string)t["StorageAccountType"],
+                                ExcludeFromLatest = (bool?)t["ExcludeFromLatest"],
```
Comment (Nickcandy) [inline src/Compute/Compute/Generated/GalleryImageVersion/GalleryImageVersionCreateOrUpdateMethod.cs:128]: Could you add test in these files: https://github.com/Azure/azure-powershell/tree/main/src/Compute/Compute/Generated/GalleryImageVersion

File: src/Compute/Compute/ChangeLog.md (hunk 1 above)
Comment (grizzlytheodore) [reply src/Compute/Compute/ChangeLog.md:35]: done

File: src/Compute/Compute/Generated/GalleryImageVersion/GalleryImageVersionCreateOrUpdateMethod.cs (hunk 2 above)
Comment (grizzlytheodore) [reply src/Compute/Compute/Generated/GalleryImageVersion/GalleryImageVersionCreateOrUpdateMethod.cs:128]: addressed

**Comment by grizzlytheodore** [comment]
@Nickcandy  i've addressed your comments. will you review merge ?

**Comment by Nickcandy** [comment]
@grizzlytheodore Sure, it looks good to me and let's wait for the ci check

**Comment by grizzlytheodore** [comment]
@Nickcandy  CI looks good!

**Pull request merged**